shared/src/framework/http/types/http-method.type.ts

```diff
@@ -1,3 +1,3 @@
-type HttpMethod = 'GET' | 'POST';
+type HttpMethod = 'GET' | 'POST' | 'PUT' | 'DELETE';
```

Comment:
I would create ENUM instead of type

Reply:
This type was in starter. I didn't change it to enum right now because it can break other students code. I suggest we should change it later